lib/protobuf/decoder.ex

```diff
@@ -115,6 +115,18 @@ defmodule Protobuf.Decoder do
       wire_64bits() ->
         <<_skip::bits-64, rest::bits>> = rest
         skip_field(rest, message, props, groups)
+
+      wire_type ->
+        message =
+          case props.field_props do
+            %{^field_number => %FieldProps{wire_type: expected, name: field}} ->
+              "field #{field}: got #{wire_type}, expected #{expected}"
+
+            _ ->
+              "field_number #{field_number}: got #{wire_type}"
+          end
+
+        raise DecodeError, message: "invalid wire_type for skipped " <> message
```

Comment:
That's great! I'd also update the error message on line 174 below, replacing _wrong_ with _invalid_, to keep it consistent.

Reply:
I think `wrong` is fair in that case - this catches a wire_type which **is** actually invalid - google protobuf only supports wire_type 1..5, and this catch 6 or 7. On line 174, that only catches when the wire_type doesn't match up with expected wire_type - it's wrong, but not necessarily invalid!